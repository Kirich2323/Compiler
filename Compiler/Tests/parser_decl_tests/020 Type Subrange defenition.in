type
    mySubrangeType = 1..2;