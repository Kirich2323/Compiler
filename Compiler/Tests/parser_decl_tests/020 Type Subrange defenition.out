main
----------
char                     type 
float                    type 
integer                  type 
mySubrangeType     type alias   subrange 1..2